Add description about steps to translate product attribute and attribute options in Magento 2 (#1433)
magento/merchdocs: src/catalog/product-translate.md

1. When complete, click <span class="btn">Save Category</span>.

1. Repeat the process for all categories used in the store.

## Step 4: Translate product attributes and attributes options

> 1. Navigate to _Stores > Attributes > Product_ and select the attribute to be translated.

Reviewer: ```suggestion
1. On. the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.

1. Select the attribute to be translated.
```

Author: Done ✅

> 1. Under the **Manage Label** section, set the translated attribute title.

Reviewer: ```suggestion
1. Choose **Manage Label** on the left and set the **Managed Titles** options to define the attribute title translations.
```

Author: Done ✅

    ![]({% link images/images/manage-label-tab.png %}){: .zoom}

>     _Manage Label tab_

Reviewer: ```suggestion
    _Manage Titles_
```

Author: Done ✅

> 1. Under _Properties > Manage Options (Values of Your Attribute)_, translate the attribute options

Reviewer: ```suggestion
1. Choose **Properties** on the left and enter the translated attribute options in the **Manage Options** section.
```

Author: Done ✅

    ![]({% link images/images/manage-option-tab.png %}){: .zoom}

>     _Manage Option tab_

Reviewer: ```suggestion
    _Manage Options_
```

Author: Done ✅

1. When complete, click <span class="btn">Save Category</span>.

1. Repeat the process for all categories used in the store.

## Step 4: Translate product attributes and attributes options

> 1. On. the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.

Reviewer: This was my typo in comments
```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
```
